order/buyer/issue/README.md

```diff
@@ -103,9 +106,22 @@ The `purchaseOrderNumber` must not contain whitespace characters.
 * `supplierContact.email`: if known, the supplier employee responsible for this order. The user with this email address should be active in Tradecloud.
 
 {% hint style="warning" %}
-`supplierAccountNumber`, `purchaseOrderNumber`, `destination.code` and `contact.email` should be unique within your company and never change. Never renumber or re-use numbers or code's.
+`id`, `supplierAccountNumber`, `purchaseOrderNumber`, `destination.code` and `contact.email` should be unique within your company and never change. Never renumber or re-use identifiers, numbers or code's.
 {% endhint %}
 
+### Company party
+
+The optional buyer, buyer accounting or supplier company party:
+
+* `id`: the optional identifier for the party, for example a GLN.
+* `idScheme`: the optional scheme, providing context to the identifier. Eg. KvK, GLN, etc.
```

Comment:
Nit, I'd formulate it like this, to make the distinction with `id` more clear. But it's minor.
```suggestion
* `idScheme`: the optional scheme, providing context to the identifier. Eg. `"KvK"`, `"GLN"`, etc.
```